pages/badges/badge-single.jsx

```diff
@@ -0,0 +1,449 @@
+      </div>
+    );
+  },
+
+  renderAchieved: function() {
+    var badgeCriteria = this.formBadgeCriteria(this.state.badge.criteria);
+    // FIXME: TODO: retrieve the information on when/how this badge was earned.
+    //              ... IF we use this information at all.
+    return (
+      <div className="badge-achieved">
+        <h3 className={'text-light'}>Congrats, you were awarded this credential.</h3>
+        <SocialShare />
+        <div className="badge-reward-text">
+          <div className="date">
+            DATE FROM API (although we may not end up using this);
+          </div>
+          <div className="qualifications">
+            EVIDENCE FOR THIS BADGE, FROM API (although we may not end up using this)
+          </div>
+        </div>
+      </div>
+    );
+  },
+
+  renderPending: function() {
+    var badgeCriteria = this.formBadgeCriteria(this.state.badge.criteria);
+    return (
+      <div className="badge-pending">
+        <h3 className={'text-light'}>Your badge claim is pending.</h3>
+        <SocialShare />
+        <div className="badge-reward-text">
+          <div className="date">
+            DATE FROM API (although we may not end up using this);
+          </div>
+          <div className="qualifications">
+            EVIDENCE FOR THIS BADGE, FROM API (although we may not end up using this)
+          </div>
+        </div>
+      </div>
+    );
+  },
+
+
+  renderEligible: function() {
+    var badgeCriteria = this.formBadgeCriteria(this.state.badge.criteria);
+    return (
+      <div className="badge-available">
+        { badgeCriteria }
+        { this.renderApplicationForm() }
+      </div>
+    );
+  },
+
+  formBadgeCriteria: function(list) {
+    // FIXME: TODO: make this a local component
+    return (
+      <div className="badge-requirement" key={''}>
+        <h3 className={'text-light'}>Badge Requirements</h3>
+        <p>Make or write something that demonstrates your understanding of any two or more of the following:</p>
+        <RequirementsList list={list} icon="fa fa-check"/>
+      </div>
+    );
+  },
+
+  renderApplicationForm: function() {
+    var showButton = this.state.evidenceText || this.state.evidenceLink || this.state.evidenceFiles.length > 0;
+
+    if (this.state.applying && this.state.showApplyModal) {
+      return (
+        <Modal modalTitle="" className="modal-credly folded" hideModal={this.hideApplyModal}>
+          <h3 className="centered">Thanks for applying for this badge!</h3>
+          <p>
+            We will be reviewing your badge application and evidence as soon as possible.
+          </p>
+          <input type="submit" className="btn center-block" onClick={this.hideApplyModal} value="Back to my badge"/>
+        </Modal>
+      );
+    }
+
+    return (
+      <div className="apply-send-qualifications">
+        <h3 className={'text-light'}>Apply for this badge</h3>
+
+        <div className="horizontal-form">
```

Comment:
it's not an HTML form, though, we never submit it. `<form>` is for when you have a real url that you post to.